### ISSUE TITLE
Bug: Linking stops after double entry in list of traceable items

### ISSUE THREAD

**Issue by bwi-mlx** [melexis/sphinx-traceability-extension#54]
If a list of linked items contains double entries (could happen by accident), the items after the double entry are not linked.

Example:
```
.. item:: SOME_DESIGN_ITEM Some design item
    :fulfills:
        SWRQT-MLX1
        SWRQT-MLX2
        SWRQT-MLX2
        SWRQT-MLX3
        SWRQT-MLX4
```

SWRQT-MLX2 is twice in the list. Because of this, links from and to SWRQT-MLX3 and SWRQT-MLX4 are not rendered.

**Comment by bwi-mlx** [comment]
Apparently the plugin reports a warning for duplicate entries, so I should have seen it, still I overlooked it.

Anyway, I think it still makes sense to take all links in the list, regardless of duplicate entries?

**Comment by SteinHeselmans** [comment]
Released in V2.5.2
